### PR TITLE
Fixed a couple of build issues.

### PR DESCRIPTION
**Pull request by justindongqiang**
Fix a couple of build issues.

### CODE REVIEW

File: sources/Applications/Web/Angular/Apps.Web.Angular.csproj [hunk 1]
```diff
@@ -38,8 +39,7 @@
   <Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish">
     <!-- As part of publishing, ensure the JS resources are freshly built in production mode -->
     <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />
-    <Exec WorkingDirectory="$(SpaRoot)" Command="npm run build -- --prod" />
```
Comment (rafsanulhasan) [inline sources/Applications/Web/Angular/Apps.Web.Angular.csproj:41]: Please keep this line to make SSR supported without the -- --prod param

File: sources/Applications/Web/Angular/Apps.Web.Angular.csproj [hunk 2]
```diff
@@ -38,8 +39,7 @@
   <Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish">
```
Comment (rafsanulhasan) [inline sources/Applications/Web/Angular/Apps.Web.Angular.csproj:39]: Add a condition that indicates the build configuration as debug/release in this way:
<Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish" Condition="'$(Configuration)'=='Release'">

File: sources/Applications/Web/Angular/Apps.Web.Angular.csproj [hunk 3]
```diff
@@ -15,6 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.SpaServices" Version="3.1.4" />
     <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="3.1.4" />
   </ItemGroup>
```
Comment (rafsanulhasan) [inline sources/Applications/Web/Angular/Apps.Web.Angular.csproj:21]: Add another ItemGroup like 
<Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish" Condition="'$(Configuration)'=='Debug'">

Comment (justindongqiang) [reply sources/Applications/Web/Angular/Apps.Web.Angular.csproj:21]: Something like this. Right?
 
` <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.SpaServices" Version="3.1.4" />
  </ItemGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="3.1.4" />
  </ItemGroup>`

**Comment by justindongqiang** [comment]
@rafsanulhasan I added changes per review. Please let me know if you have any questions.

**Pull request merged**